testing/load-testing/README.md

```diff
@@ -0,0 +1,54 @@
+# K6
+
+K6 is a load testing tool that uses JavaScript and Go to fake multiple simultaneous requests to an API or Web Server.
+It gives instant information of the requests results and allows you to easily test the performance limits of your API.
+
+## Installing
+
+### Mac
+
+```
+brew tap loadimpact/k6
+brew install k6
+```
+
+### Docker
+
+```
+docker pull loadimpact/k6
+```
+
+### Linux
+
+[Download](https://github.com/loadimpact/k6/releases) and extract the latest version.
+
+## Usage
+
+### What you shall test and how do you report it
+
+In the following [Document](https://docs.google.com/document/d/1izTKBOYPYUuc8zmxmvB4EQFbhMDgaDvYjViDL_yDvOg/edit?ts=59382e24#) it's explained how and what things you shall test. Also it tells you how to report your results and gives you the templates where to put them.
```

Comment:
I have no access to this document. I'd also change it with a md file inside this repository so that we have all files needed in this repo. What do you think!?